app/controllers/downtimes_controller.rb

```diff
@@ -39,7 +39,7 @@ def edit
   def update
     @downtime = Downtime.for(@edition.artefact)
 
-    if params["commit"] == "Cancel downtime"
+    if !params[:downtime]
```

Comment:
We're going to need to change the `update` method to do the same extra validation that is done in the `create` method now.

(We should probably add some test coverage around this.)